tests/CascadeTest.php

```diff
@@ -70,8 +71,8 @@ public function testGetLoggerWithInheritance()
     public function testGetLoggerWithInheritanceAndNonAlphanumericParentNames()
     {
         Cascade::setInheritOnUndefined(true);
-        $parent = Cascade::getLogger('logger_A');
-        $child = Cascade::getLogger('logger_A.child_A');
+        $parent = Cascade::getLogger('logger_Aé');
```

Comment:
Looks like you're using UTF-8 chars for this one? Any reason why? Or is this a strange encoding display?